packages/syft/src/syft/service/worker/worker_pool.py

```diff
@@ -79,11 +107,76 @@ class WorkerPool(SyftObject):
 
     __attr_unique__ = ["name"]
     __attr_searchable__ = ["name", "syft_worker_image_id"]
+    __repr_attrs__ = [
+        "name",
+        "syft_worker_image_id",
+        "max_count",
+        "workers",
+        "created_at",
+    ]
 
     name: str
     syft_worker_image_id: UID
+    syft_worker_image_name_tag: Optional[str]
     max_count: int
     workers: List[SyftWorker]
+    created_at: DateTime = DateTime.now()
+
+    @property
+    def running_workers(self) -> List[SyftWorker]:
+        return [
+            worker
+            for worker in self.workers
+            if get_worker_container_status(worker) == WorkerStatus.RUNNING
```

Comment:
This would create multiple `DockerClient`s, one for each worker, but don't worry about this I'll push a fix for this.